app/models/section.rb

```diff
@@ -159,14 +160,18 @@ def eql?(other)
     id.eql?(other.id)
   end
 
-protected
-
-  attr_reader :slug_generator, :edition_factory
-
   def never_published?
     !published?
   end
 
+  def change_note_required?
+    !(never_published? || minor_update?)
```

Comment:
I wonder whether this would be a bit clearer written as `published? && !minor_update?`?